docs/semgrep-code/overview.md

```diff
@@ -33,23 +35,44 @@ Semgrep Cloud Platform (SCP) displays Semgrep Code's findings. Additionally, SCP
 * Customize how Semgrep Code scans your repositories
 * Manage your users and facilitate team collaboration in remediating security issues
 
-## OSS versus Pro Engine
+## OSS versus Semgrep Code analysis
 
-By default, Semgrep Code is powered by Semgrep's OSS engine. It can analyze interactions within a single function, a process known as intraprocedural analysis, and its smaller scope of analysis makes it faster and easier to integrate into developer workflows.
+By default, Semgrep Code can analyze interactions beyond a single function but within a single file, a process known as **cross-function or interprocedural analysis**. This smaller scope of analysis makes it faster and easier to integrate into developer workflows.
 
-For cross-file and cross-function analysis (interfile and interprocedural analysis, respectively), you can enable [Semgrep's Pro Engine](/semgrep-code/semgrep-pro-engine-intro/). These scans produce fewer false positives and more true positives, but take longer to complete.
+Semgrep OSS can only analyze interactions within a single function, known as intraprocedural or single-function analysis. Semgrep Code scans take slightly longer to finish. <!-- can we have a stat for this -->
```

Comment:
```suggestion
Semgrep OSS can only analyze interactions within a single function, known as intraprocedural or single-function analysis. However, this means that Semgrep OSS is slightly faster than Semgrep Code. <!-- can we have a stat for this -->
```

Reworded this slightly, since this paragraph is focused on OSS.

Reply:
I don't have a stat at the ready :( but need to work on this!